schemas/webhooks/definitions.yaml

```diff
@@ -53,6 +53,15 @@ webhook_update:
     url:
       $ref: "#/webhook_url"
 
+webhook_event_resource_type:
```

Comment:
We have more resource types I believe

Reply:
Ah yeah, we're probably missing `watchlist_monitor`. Is that correct?

Reply:
Correct